vault-operator/README.md

```diff
@@ -106,6 +106,19 @@ For example:
 
 `kubectl -n default get vault example ...` -> `kubectl -n default get vaultservice example ...`
 
+## Tests
+This repo contains some tests that use the operator-sdk's test framework. These tests are based directly on the original vault-operator
+tests, and thus cannot fully complete when run on a local machine and must be run inside a kubernetes cluster instead. This is a very
```

Comment:
Can you bold this sentence `thus cannot fully complete when run on a local machine and must be run inside a kubernetes cluster instead` so that the user knows that i can't be run locally.